packages/synapse-react-client/src/lib/utils/hooks/useLogin.ts

```diff
@@ -26,6 +28,7 @@ export type UseLoginReturn = {
     otpType?: TwoFactorAuthOtpType,
   ) => Promise<void>
   errorMessage: string | undefined
+  isLoading: boolean
```

Comment:
I noticed Adam changed the button text/disabled status in the designs. It makes the login components feel much more responsive